src/probes/crash_probe.c

```diff
@@ -427,12 +427,13 @@ static bool in_clr_build(char *fullpath)
          * The build environment for Clear Linux packages is set up by 'mock',
          * and the chroot in which rpmbuild builds the packages has this prefix.
          */
-        if (g_regex_match_simple("^[!/]builddir[!/]build[!/]BUILD[!/]",
-                                 fullpath, 0, 0) == TRUE) {
-                return true;
-        }
+	if (strstr(fullpath,"/builddir/build/BUILD/"))
```

Comment:
More robust would be to use `strncmp` here as well, but because the pattern is highly unlikely to appear in a full path other than the beginning, I would accept this change.